pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -32,18 +32,6 @@ func NewResourceSyncController(
 	); err != nil {
 		return nil, err
 	}
-	if err := resourceSyncController.SyncConfigMap(
-		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespace, Name: "aggregator-client-ca"},
-		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalMachineSpecifiedConfigNamespace, Name: "aggregator-client-ca"},
-	); err != nil {
-		return nil, err
-	}
-	if err := resourceSyncController.SyncConfigMap(
-		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.OperatorNamespace, Name: "initial-client-ca"},
```

Comment:
this ca isn't under management and we have no need for an intermediate copy anymore (we have a multi-namespace configmaplister now)